api_app/core/cloud.py

```diff
@@ -0,0 +1,13 @@
+from msrestazure.azure_cloud import AZURE_PUBLIC_CLOUD, AZURE_US_GOV_CLOUD
+from core import config
+
+
+def get_cloud():
```

Comment:
Aren't you missing some of valid values?
What will happen if a match isn't found?
Should we do a case insensitive comparison? 
Is it possible to do a map instead of multiple if statements?

Reply:
Do we want to note we support other values/clouds? I am not sure, we know already things are different on each cloud... So don't want to confuse ... The arm environment param is validated as part of the config schema validation , can contain only these values .. Will add the other checks you mentioned..

Reply:
perhaps we could raise an error if we don't support the cloud yet,
but if we don't support certain clouds maybe we should remove them from the schema of the config.yaml?

Reply:
Currently our config allows all cloud types

Reply:
Thinking now we better remove it from there too, WDYT? added the change, let me know in case you disagree